DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -98,8 +101,20 @@ function PreFirstEntryMapFixes()
             foreach AllActors(class'#var(prefix)BlackHelicopter',jock,'Helicopter'){break;}
             hoverHint = class'DXRTeleporterHoverHint'.static.Create(self, "", jock.Location, jock.CollisionRadius+5, jock.CollisionHeight+5, exit.Name);
             hoverHint.SetBaseActor(jock);
+        }
 
+        foreach AllActors(class'#var(prefix)SecurityBot2',sb,'enemy_bot') {
+            hoverHint = class'DXRHoverHint'.static.Create(self, "MJ12 Security Bot " $ ++botNum, sb.Location, sb.CollisionRadius*1.1, sb.CollisionHeight*1.1, sb.Name);
```

Comment:
```suggestion
            hoverHint = class'DXRHoverHint'.static.Create(self, "MJ12 Security Bot " $ ++botNum, sb.Location, sb.CollisionRadius*1.11, sb.CollisionHeight*1.11, sb.Name);
```

is  `* 1.1` enough? DXREnemies can make them up to 1.1x size, you want the hint to be bigger not equal